student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name)
+    super(age, name)
+    @color = classroom
+  end
+
+  def can_use_services?
+    true
+  end
```

Comment:
- [ ] This method should belong to the `Teacher` class instead of the `Student` class. Kindly fix that.

Reply:
Thanks!